packages/paste-core/primitives/sibling-box/__tests__/siblingBox.test.tsx

```diff
@@ -60,4 +65,43 @@ describe('SiblingBox render', () => {
       .toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  describe('HTML attributes', () => {
+    it('should set a default element data attribute', () => {
```

Comment:
Why did you add the `: void` return type on the next 2 `it` blocks but not this one?

Reply:
Copy paste... :hide: